test/fixtures.rb

```diff
@@ -107,11 +107,11 @@ def call(params)
   end
 end
 
-class MyCustomError < StandardError ; end
```

Comment:
@jodosha reverting the change for `ErrorCallWithStringMethodNameAsHandlerAction` will result in a failing test in `action_test.rb`, this one 
```
it 'handles exception with specified method (string)' do
  response = ErrorCallWithStringMethodNameAsHandlerAction.new.call({})

  response[0].must_equal 502
  response[2].must_equal ['Lotus::Controller::Error']
end
```

unless I revert the change there as well, please advice, thanks.